src/benchmark_register.cc

```diff
@@ -162,8 +163,15 @@ bool BenchmarkFamilies::FindBenchmarks(
                   StringPrintF("%s:", family->arg_names_[arg_i].c_str());
             }
           }
-
+          
+#if defined(__ANDROID__) && (defined(__GLIBCXX__) || defined(__GLIBCPP__))
```

Comment:
`__GLIBCPP__` was removed in favor of `__GLIBCXX__` in GCC 3.4. I can't imagine anybody is still using that old of a GCC version. Are you sure it's needed?